ci-operator/step-registry/assisted/common/verify/assisted-common-verify-commands.sh

```diff
@@ -29,17 +29,12 @@ echo "### Copying test-list file"
 scp -F ${SHARED_DIR}/ssh_config "${SHARED_DIR}/test-list" "root@ci_machine:/tmp/test-list"
 
 echo "### Running tests"
-timeout --kill-after 10m 120m ssh -F ${SHARED_DIR}/ssh_config "root@ci_machine" bash - << EOF
+timeout --kill-after 5m 120m ssh -F ${SHARED_DIR}/ssh_config "root@ci_machine" bash - << EOF
```

Comment:
I changed this grace-period from 10m to 5m because the total grace-period of this job is 10m and we'd like to leave some time to gathering the xunit XMLs and logs